### DISCUSSION TITLE
Wiki dead?

### DISCUSSION THREAD

**Discussion by darthvader45** [NovaMachina-Mods/ExNihiloSequentia#415] [General]
Decided to look up Ex Nihilo to see how far the mod has come, and stumbled across this gem.  Decided to take a look at the wiki and.....couldn't connect to the site.  Is it dead?  I'd love to have more knowledge of this new iteration, so as to be able to get a fully informed start.

**Comment by NovaMachina** [comment]
It's been moved. I haven't updated all the wiki links. You can find it here now: novamachina-mods.github.io

**Reply by darthvader45** [reply]
Thanks.
